src/frequenz/sdk/actor/_actor.py

```diff
@@ -58,19 +62,14 @@ def start(self) -> None:
     async def _run(self) -> None:
         """Run this actor's logic."""
 
-    async def _delay_if_restart(self, iteration: int) -> None:
-        """Delay the restart of this actor's n'th iteration.
-
-        Args:
-            iteration: The current iteration of the restart.
-        """
+    async def _delay(self) -> None:
```

Comment:
Maybe make it a bit more explicit?

```suggestion
    async def _delay_restart(self) -> None:
```